types/i3dh-ios-bridge/index.d.ts

```diff
@@ -0,0 +1,86 @@
+// Type definitions for non-npm package i3dh-ios-bridge 1.0
+// Project: https://git.threedy.io/misc/realitybenchmark_suite/
+// Definitions by: Tim Lâm <https://github.com/TimGeronimoLam>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export namespace i3dh {
+    enum ResolutionProfile {
```

Comment:
Please rethink if you need `enums` here, maybe const would suffice:
https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums
(of course, I cannot see the original JS code usage/symbols here)

Reply:
This enum is used for consistency because the rest of the codebase also uses enums.